### ISSUE TITLE
Predict func crash the kernel and the program CatBoostRegressor.predict(<empty_data_frame>)

### ISSUE THREAD

**Issue by snirbenyosef** [catboost/catboost#1385]
Problem:
#### Describe the bug
I have a problem with predict func .
my dataframe Z_jor[something] is empty. And I try to use it in the predict func. I want it to continue but it just reset my kernel on jupiter notebook. I also  run my code from shell I get this error:
```python
No objects info loaded 
Floating point exception (core dumped)
```
#### Steps/Code to Reproduce

Example:
```python
import pandas as pd
import numpy as np
import pickle
import warnings
import matplotlib.pyplot as plt
import os
import datetime
from sklearn.preprocessing import LabelEncoder
from catboost import Pool, CatBoostRegressor

model_path = my model path
z_jor = some data frame 

regressor = pickle.load(open(model_path, 'rb'))['model']

regressor.predict(z_jor[something]) ## this crash the kernel
```

regressor:
```
print(regressor)
<catboost.core.CatBoostRegressor object at 0x7f689e3dcc18>
```

#### Expected Results
Not Crashing the kernel or show warrning  or clear error

#### Actual Results
1) from jupiter notebook>> just restart the kernel
2) from shell >>
```python
No objects info loaded 
Floating point exception (core dumped)
```

#### Versions
```
System:
    python: 3.6.9 (default, Jul 17 2020, 12:50:27)  [GCC 8.4.0]
executable: /usr/bin/python3
   machine: Linux-4.15.0-1083-azure-x86_64-with-Ubuntu-18.04-bionic

Python dependencies:
          pip: 20.1.1
   setuptools: 49.2.0
      sklearn: 0.23.1
        numpy: 1.19.1
        scipy: 1.5.2
       Cython: None
       pandas: 1.0.5
   matplotlib: 3.3.0
       joblib: 0.16.0
threadpoolctl: 2.1.0

```

catboost version: 0.23.2
Operating System: Linux-4.15.0-1083-azure-x86_64-with-Ubuntu-18.04-bionic
CPU: 8
# GPU:0

**Comment by Sorkanius** [comment]
Can confirm this happens.

**Comment by ibobak** [comment]
I have exactly the same problem now with the last version of catboost. Also loaded the model (previously trained and saved as cbm).  On one machine I am getting 
Floating point exception (core dumped)

on another machine (the same code, same model) I am getting

Process finished with exit code 136 (interrupted by signal 8: SIGFPE)

Please, help.

**Comment by LyzhinIvan** [comment]
Fixed by ad78255c143ebe82b7f94428abea3417b9d60033
